src/bin/index.ts

```diff
@@ -147,14 +148,22 @@ async function run(args: CliArgs) {
   // lint package
   await lint(cwd)
 
+  const spinner = ora({
+    text: 'bundling...\n\n',
+    spinner: 'dots',
+    color: 'green',
+  })
   try {
+    spinner.start()
```

Comment:
We could use [oraPromise](https://github.com/sindresorhus/ora?tab=readme-ov-file#orapromiseaction-text), but it outputs a green checkmark where it can't be opted-out by default.